packages/nextjs/app/builders/0x97DB651f21639C4F6b0997184373b9d21179C157/page.tsx

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable react/no-unescaped-entities */
+
+/* eslint-disable @next/next/no-img-element */
+import Image from "next/image";
+import { BuidlGuidlLogo } from "~~/components/assets/BuidlGuidlLogo";
+
+export default function PersonalPage() {
+  return (
+    <div className="flex-grow bg-base-300 w-full mt-16 px-8 py-12">
+      <div className="flex bg-base-100 px-10 py-10 items-center rounded-3xl  mb-4 md:mb-0">
+        <div className="flex flex-col md:w-1/2">
+          <Image src="/assets/Sahil.png " alt="Sahil Kharya" width={181} height={181} className="rounded-full" />
```

Comment:
You should remove the empty space at the end of the file name, you image is not visible on the preview page 

`<Image src="/assets/Sahil.png "...`